.github/workflows/bcr/presubmit.yml

```diff
@@ -6,7 +6,5 @@ bcr_test_module:
     run_tests:
       name: "Run test module"
       platform: ${{ platform }}
-      test_flags:
-      - "--enable_runfiles"
```

Comment:
I added this yesterday to match what's already in the bcr, but we shouldn't need this as runfiles is already enabled in our `.bazelrc`.